Test-№1.1-from-19.10.2022/task1.c

```diff
@@ -0,0 +1,76 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#define constSizeStr 10
+
+bool isPalindrome(char *string, int length)
+{
+    bool soFarFit = true;
+    for (int i = 0; i < length / 2; ++i)
+    {
+        if (string[i] != string[length - 1 - i])
+        {
+            soFarFit = false;
+            break;
+        }
+    }
+    return soFarFit;

```

Comment:
```suggestion
    for (int i = 0; i < length / 2; ++i)
    {
        if (string[i] != string[length - 1 - i])
        {
            return false;
        }
    }
    return true;

```
Лишняя переменная `soFarFit`. Если в цикле доказываем, что входная строка не является палиндромом - возвращаем `false`. Если не доказываем, то после цикла возвращаем `true`.